Birthdays/Birthdays/Program.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Birthdays
+{
+    class Program
+    {
+        public static Dictionary<String, List<string>> dictionary = new Dictionary<string, List<string>>();
+        static void Main()
+        {
+            var month = new int[12];
+
+            try
+            {
+                using(StreamReader sr = new StreamReader("birthday.txt"))
+                {
+                    string line;
+                    while((line = sr.ReadLine()) != null)
+                    {
+                        ParseLineString(line);
+                    }
+                }
+            }
+            catch { }
+
+            while (true)
+            {
+                Console.WriteLine("Введите имя");
+                var name = Console.ReadLine();
```

Comment:
Все имена в файле данных записаны строчными буквами. Пользователь может ввести с консоли (и, скорее всего, так и сделает) имя с прописной буквы. Нужно привести имя к нижнему регистру
```suggestion
                var name = Console.ReadLine().ToLower();
```